REPL.h

```diff
@@ -66,6 +70,7 @@ struct REPL
         }
     };
 
+    bool m_is_playground;
```

Comment:
Can state mutate?  Why not make this `const`?  It shouldn't change after construction right?